### PR TITLE
Adding arch linux packages to README.md

### PR DESCRIPTION
**Pull request by frealgagu**
This commit contains information about ArchLinux packages in README.md.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,6 +27,20 @@ sudo apt-get update
 sudo apt-get install lazygit
 ```
 
+### Arch Linux
+Packages for Arch Linux are available via AUR (Arch User Repository).
+
+There are two packages. The stable which is built with the last releases and the git version which takes the last commit and build based on.
```
Comment (jesseduffield) [inline README.md]: there are some grammar errors here, and I think it's missing a few words in the second sentence?

**Comment by frealgagu** [comment]
Are you OK if I put:

### Arch Linux
Packages for Arch Linux are available via AUR (Arch User Repository).

There are two packages. The stable one which is built with the last releases and the git version which takes the last commit and build based on it.

**Comment by jesseduffield** [comment]
All good! I've made a couple changes:

### Arch Linux
Packages for Arch Linux are available via AUR (Arch User Repository).

There are two packages. The stable one which is built with the latest release and the git version which builds from the most recent commit.

**Comment by jesseduffield** [comment]
looks good!

**Pull request merged**